.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "airbnb/base",
```

Comment:
ESLint is failed to load in your project - it is not working!
Here is the problem:

![image](https://user-images.githubusercontent.com/31325372/94987571-ebc50b00-056f-11eb-9f05-7cb43284e5c3.png)

(btw, `airbnb/base` is deprected - [docs](https://www.npmjs.com/package/eslint-config-airbnb#eslint-config-airbnbbase))

Hence, you don't have linting in your project at all!

Here is the `CardFilm` component with enabled linting:

![image](https://user-images.githubusercontent.com/31325372/94987611-334b9700-0570-11eb-98ce-dcda91e417cb.png)

Hooks are used incorrectly, exhaustive deps array is missing necessary variables

`FormInfo` component:

![image](https://user-images.githubusercontent.com/31325372/94987676-ba990a80-0570-11eb-9ce3-0b506e1fe9c7.png)